frontend/src/components/AppFooter.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <footer>
+    <a class="footer-link" href="https://github.com/uab-cgds-worthey/rosalution/blob/main/COPYING">
+      Copyleft <span class="copy-left">&copy;</span> {{ currentYear }} Center for Computational Genomics and Data
```

Comment:
Hmmm, we didn't discuss including the copyleft license in the footer.  This is really interesting.  Have you seen this done somewhere before?  I was just imaging the name of the center, and the fully qualified name of the schoool of medicine.